x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/type/EsqlDataTypes.java

```diff
@@ -52,15 +51,6 @@ public final class EsqlDataTypes {
         ES_TO_TYPE = Collections.unmodifiableMap(map);
     }
 
-    private static final Map<String, DataType> NAME_OR_ALIAS_TO_TYPE;
```

Comment:
Not sure about the move, there is some other bits of related functionality in this class (`NAME_TO_TYPE`, but also `ES_TO_TYPE` and other "to type" resolvers).

Reply:
I think we're in the process of moving it all. I just grabbed the ones I was touching.

Reply:
I'll get this in and then open a follow up PR to move the rest.

Reply:
Well, I'm going to move the methods one at a time. The first one I looked at was subtly different so I figure that's safer.

Reply:
> Well, I'm going to move the methods one at a time.

Sounds good. As long as we're converging to a consistent state.